--- abe/configure.in.link	2011-02-02 08:21:43.000000000 +0100
+++ abe/configure.in	2011-02-02 08:22:46.000000000 +0100
@@ -99,17 +99,6 @@
 fi
 
 dnl --------------------------------------------------------------------------
-dnl Check for X  
-dnl Note that we only need libs (so that glut links properly on all systems).
-dnl --------------------------------------------------------------------------
-if test "x$is_win32" = "xno" ; then
-    AC_PATH_XTRA
-    if test "x$no_x" = "x" ; then
-	TR_LIBS="$X_PRE_LIBS $X_LIBS -lX11 -lXi -lXext -lXmu -lXt $X_EXTRA_LIBS $TR_LIBS"
-    fi
-fi
-
-dnl --------------------------------------------------------------------------
 dnl Check for some common libraries
 dnl --------------------------------------------------------------------------
 AC_CHECK_LIB( dl, main, TR_LIBS="$TR_LIBS -ldl" )
